### ISSUE TITLE
CoolProp Module not Found

### ISSUE THREAD

**Issue by AndrewPrice775** [CoolProp/CoolProp#1952]
### Description

CoolProp Module Not Found

```
(base) C:\Users\****>python
Python 3.7.7 (default, May  6 2020, 11:45:54) [MSC v.1916 64 bit (AMD64)] :: Anaconda, Inc. on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> import CoolProp
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\Users\****\anaconda3\lib\site-packages\CoolProp\__init__.py", line 16, in <module>
    from .CoolProp import AbstractState
ImportError: DLL load failed: The specified module could not be found.
```

### Steps to Reproduce

1. Installed Anaconda
2. pip install CoolProp
3. No errors
4. Open Anaconda Prompt. 
5. Type Python
6. Type Import CoolProp
7. Receive Error, Module Not Found.

**Expected behavior:** [What you expect to happen]
:)  Happiness that I got it to work.

**Actual behavior:** [What actually happens]
:( Sadness because I failed.

### Versions

**CoolProp Version:** 6.4.0
**Operating System and Version:** Windows 10 Home, 64-bit
**Access Method:** 
Anaconda -V = Anaconda Version 1.7.2
Python -V  = 3.7.7

### Additional Information

I'm new to Python. For what it is worth, I have gotten the CoolProp Excel Wrapper to work.

**Comment by AndrewPrice775** [comment]
(base) C:\Users\****>pip show --verbose CoolProp
Name: CoolProp
Version: 6.4.0
Summary: Open-source thermodynamic and transport properties database
Home-page: http://www.coolprop.org
Author: Ian Bell
Author-email: ian.h.bell@gmail.com
License: UNKNOWN
Location: c:\users\****\anaconda3\lib\site-packages
Requires:
Required-by:
Metadata-Version: 2.1
Installer: pip
Classifiers:
  Programming Language :: Python
  Development Status :: 4 - Beta
  Environment :: Other Environment
  Intended Audience :: Developers
  License :: OSI Approved :: MIT License
  Operating System :: OS Independent
  Topic :: Software Development :: Libraries :: Python Modules
Entry-points:

**Comment by ibell** [comment]
This is a bug introduced in 6.4.0 in our build system.  Please install the dev wheel with the third option here: http://www.coolprop.org/coolprop/wrappers/Python/index.html#automatic-installation .  The "If you dare" option.  That will install 6.4.1, which will be released as soon as we fix #1941 , and should fix this issue.

**Comment by AndrewPrice775** [comment]
Thank you very much! Working as expected.
